backend/main.go

```diff
@@ -57,6 +60,36 @@ func main() {
 		}
 		ruleDataList = append(ruleDataList, rd)
 	}
+
+	for {
+		if _, err := control.GetScanStatus(); err != nil {
+			time.Sleep(5 * time.Second)
```

Comment:
Can you make a constant at the top of the file: const ScanStatusInterval = 5 * time.Second and use the constant here. Better style.